modules/interactive_music/editor/audio_stream_interactive_editor_plugin.cpp

```diff
@@ -148,6 +148,9 @@ void AudioStreamInteractiveTransitionEditor::_update_selection() {
 }
 
 void AudioStreamInteractiveTransitionEditor::_cell_selected(TreeItem *p_item, int p_column, bool p_selected) {
+	if (!p_item->has_meta("to")) {
+		return;
+	}
```

Comment:
If this line is never supposed to be reached, it might be better to use `ERR_FAIL_COND()` instead so an error can be printed.

Reply:
> If this line is never supposed to be reached, it might be better to use `ERR_FAIL_COND()` instead so an error can be printed.

It does get reached though, hence why I added it.
It's been a bit since I made this PR, so I'll see if it would instead make sense to always add that metadata.

Reply:
> > If this line is never supposed to be reached, it might be better to use `ERR_FAIL_COND()` instead so an error can be printed.
> 
> It does get reached though, hence why I added it. It's been a bit since I made this PR, so I'll see if it would instead make sense to always add that metadata.

Now I remember, "to" is an integer referencing a clip. It's not written to the corner cells of the tree (ie. the "From / To" header)
It doesn't make any sense to write it there. Still, checking if `"to"` exists is a bit specific, so instead I've made it so that each cell has an `"is_editable_cell"` metadata and check that.